sqlglot/dialects/postgres.py

```diff
@@ -613,6 +637,7 @@ class Generator(generator.Generator):
             exp.Rand: rename_func("RANDOM"),
             exp.RegexpLike: lambda self, e: self.binary(e, "~"),
             exp.RegexpILike: lambda self, e: self.binary(e, "~*"),
+            exp.Round: _round,
```

Comment:
Nit: Can we get rid of this entry and move `_round` under the `Generator` class, named as `round_sql`? I'd also inline `_to_decimal`.